packages/components/src/color-control/index.native.js

```diff
@@ -1,27 +1,31 @@
 /**
```

Comment:
Are you sure that you want to move that component instead of creating a new one? I believe that `unsupported-footer-control` could be used in the future.

Reply:
Reverted.